bedrock/firefox/views.py

```diff
@@ -591,6 +591,33 @@ def get_template_names(self):
                 template = 'firefox/whatsnew/index.html'
         elif locale == 'id':
             template = 'firefox/whatsnew/index-lite.id.html'
+        elif version.startswith('78.'):
+            if variation == '2' and locale in ['en-US', 'en-CA', 'en-GB']:
```

Comment:
There's a lot of logic here that's quite long winded. It also feels a little confusing for me to work out which template I'm seeing, when we use `?v=2` in the URL but `a` in the template name as an identifier? 

I'd suggest renaming the templates to be consistent with the variation parameter. Then we could also cut down on a lot of this logic using something like:

```python
variations = ['2', '3', '4', '5']
locales = ['en-US', 'en-CA', 'en-GB', 'de', 'fr']

if variation in variations and locale in locales:
    locale = locale.split('-')[0]
    template = 'firefox/whatsnew/whatsnew-fx78-{0}-{1}.html'.format(variation, locale)
else:
    template = 'firefox/whatsnew/index.html'
```

Adding some unit tests for the above would be nice too.